AdminWebsite/AdminWebsite/ClientApp/src/app/booking/summary/summary.component.spec.ts

```diff
@@ -22,7 +22,7 @@ import { VideoHearingsService } from '../../services/video-hearings.service';
 import { MockValues } from '../../testing/data/test-objects';
 import { BookingEditStubComponent } from '../../testing/stubs/booking-edit-stub';
 import { ParticipantsListStubComponent } from '../../testing/stubs/participant-list-stub';
-import { ParticipantsListComponent } from '../participants-list/participants-list.component';
+import { ParticipantListComponent } from '../participant/list/participant-list.component';
```

Comment:
was this the name of the file i commented on earlier?

Reply:
You're right - it is. I can shorten that import.